ArduCopter/sensors.cpp

```diff
@@ -1,5 +1,15 @@
 #include "Copter.h"
 
+void Copter::read_airspeed(void)
+{
+    if (airspeed.enabled()) {
+        airspeed.read();
+        if (should_log(MASK_LOG_IMU)) {
+            DataFlash.Log_Write_Airspeed(airspeed);
+        }
+    }
+}
```

Comment:
I can shed some light here. The airspeed sensor is usually the best source of OAT (Outside Air Temperature) as the onboard baro's experience signficant heating from the surrounding components (in fact it's severe enough we artifically clip the max reported baro temp since it has normally be heated so much). Airspeed sensors are usually installed a far distance from the rest of the avionics and as such don't experience the same level of heating as the barometers do, so we feed the airspeed temperature to the baro if possible.

If copter adds support for airspeed sensors then we might want to shift this setting to be inside one of the libraries via singletons so that the vehicle doesn't have to worry about it. At the moment though bringing in the code @bnsgeyer is pointing at would be a good idea.

Reply:
This no longer seems to be being done by Plane.  If we're going to do it , it should be up at the library level.

Reply:
@WickedShell can you clear up this confusion?

Reply:
@peterbarker https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Baro/AP_Baro.cpp#L394 got moved up in the end of 2018.